web/app/configs/lang.json

```diff
@@ -141,9 +141,17 @@
         "importfrom": "of import from",
         "productionimport": "production/import",
         "consumption": "consumption",
-        "electricityfrom": "electricity comes from",
-        "electricityto": "electricity is exported to",
-        "electricitystored": "electricity is stored using",
+
+        "electricityComesFrom": "{1} % of electricity in <img id=\"country-flag\"></img> <b>{2}</b> comes from {3}",
```

Comment:
@Raffox97 what do you think about this sentence structure for translations? It will be much easier!